src/schema/v2/viewingRoomSubsection.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { GravityARImageType } from "./GravityARImageType"
+
+export const ViewingRoomSubsectionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "ViewingRoomSubsection",
+  fields: () => {
+    return {
+      internalID: {
```

Comment:
**suggestion**: I think we can also spread IDFields in here to avoid redefining ([example](https://github.com/artsy/metaphysics/blob/d0d6ab07696d58582624b05eebb8d7e525e61e46/src/schema/v2/partnerOfferToCollector.ts#L21)).

Reply:
oh actually we don't need an `id` field here (and in [another PR](https://github.com/artsy/metaphysics/pull/6250) too) - only internalID. Object identification module doesn't have only internalID's definition but I can add it there and then spread it in these two places.

@anandaroop @dblandin do you mind if I do it separately in another PR just for my convenience?

Reply:
@nickskalkin Yep, no worries. Can be a follow-up 👍🏼